src/bsoncxx/view_or_value.hpp

```diff
@@ -123,6 +123,30 @@ class view_or_value {
     View _view;
 };
 
+///
+/// Equality operators for comparison with plain Views.
+///
+
+template <typename View, typename Value>
+BSONCXX_INLINE bool operator==(view_or_value<View, Value> lhs, View rhs) {
```

Comment:
That makes sense.  The same should be true for the operator== below where view_or_value is on the rhs, I think.